packages/loader/src/prefab/PrefabParserContext.ts

```diff
@@ -0,0 +1,9 @@
+import { Engine, Entity } from "@galacean/engine-core";
+import { ParserContext } from "../resource-deserialize/resources/parser/ParserContext";
+import { IHierarchyFile } from "../resource-deserialize";
+
+export class PrefabParserContext extends ParserContext<IHierarchyFile, Entity> {
```

Comment:
is PrefabParserContext the same as ParserContext?

Reply:
removed